src/main/resources/configs/jcontent.jsp

```diff
@@ -8,6 +8,8 @@ contextJsParameters.config.links['importAcademy']="<%= SettingsBean.getInstance(
 <%
     ConfigService configService = BundleUtils.getOsgiService(ConfigService.class, null);
     Config automatedTagConfig = configService.getConfig("org.jahia.modules.automatedtags.service.impl.AutomatedTagServiceImpl");
-    String tagImageOnUploadFlag = automatedTagConfig.getRawProperties().get("automated-tags.tagImageOnUpload");
+    String tagImageOnUploadStr = automatedTagConfig.getRawProperties().get("automated-tags.tagImageOnUpload");
+    Boolean tagImageOnUploadFlag = (tagImageOnUploadStr != null && !tagImageOnUploadStr.isEmpty() &&
```

Comment:
perhaps better to use already existing `Boolean.valueOf(tagImageOnUploadStr)` instead of manually parsing

Reply:
Thanks updated